pkg/client/unversioned/restclient.go

```diff
@@ -73,6 +84,33 @@ func NewRESTClient(baseURL *url.URL, apiVersion string, c runtime.Codec, maxQPS
 	}
 }
 
+// readExpBackoffConfig handles the internal logic of determining what the
```

Comment:
I think this should be a function that returns a backoff object, not two config parameters from it. Ideally returning NoBackoff when unset.